Pipeline/FeatureExtractor/FeatureExtraction.py

```diff
@@ -18,14 +19,48 @@ def get_audioset_features(path_to_pkl, path_to_save_pkl):
         file_list = data_dict_music.get("files_list")
         assert file_list is not None, "files list not found in "+path_to_pkl
 
-        #model definition
-        model = VGGishModel.VGGishModel()
 
-        data_dict_music["embeddings_list"] = model.get_embeddings_list(file_list)
+        data_dict_music["embeddings_list"] = FeatureExtraction.__get_features(file_list)
 
         with open(path_to_save_pkl, "wb") as handle:
             pickle.dump(data_dict_music, handle)
 
+    @staticmethod
+    def get_audioset_features_file(path_to_file, path_to_save_pkl):
+        assert isinstance(path_to_file, str) and (path_to_file.endswith('.wav') or os.path.isdir(path_to_file)),\
```

Comment:
'And' in Python has higher priority than 'or' but just in case, please, make in more clear and use parentheses

Reply:
I didn't understand the comment, because 'or' is already wrapped in parentheses

Reply:
I'm sorry for the comment - my fault, I didn't notice